stdlib/Sockets/src/Sockets.jl

```diff
@@ -625,7 +625,7 @@ listen(port::Integer; backlog::Integer=BACKLOG_DEFAULT) = listen(localhost, port
 listen(host::IPAddr, port::Integer; backlog::Integer=BACKLOG_DEFAULT) = listen(InetAddr(host, port); backlog=backlog)
 
 function listen(sock::LibuvServer; backlog::Integer=BACKLOG_DEFAULT)
-    uv_error("listen", trylisten(sock))
+    uv_error("listen", trylisten(sock; backlog=backlog))
```

Comment:
Can also use the shorter
```suggestion
    uv_error("listen", trylisten(sock; backlog))
```